test-packages/platform-feature-tests/cypress/support/flows/sections/lpaManageAppeals/houseHolderQuestionnaire.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable cypress/unsafe-to-chain-command */
+// @ts-nocheck
+/// <reference types="cypress"/>
+
+import { BasePage } from "../../../../page-objects/base-page";
+import { AppealProcess } from "../../pages/lpa-manage-appeals/appealProcess";
+import { ConstraintsAndDesignations } from "../../pages/lpa-manage-appeals/constraintsAndDesignations";
+import { ConsultResponseAndRepresent } from "../../pages/lpa-manage-appeals/consultResponseAndRepresent";
+import { NotifyParties } from "../../pages/lpa-manage-appeals/notifyParties";
+import { PoReportAndSupportDocs } from "../../pages/lpa-manage-appeals/poReportAndSupportDocs";
+import { SiteAccess } from "../../pages/lpa-manage-appeals/siteAccess";
+
+export const householderQuestionnaire = (context, lpaManageAppealsData) => {
+	const basePage = new BasePage();
+	const constraintsAndDesignations = new ConstraintsAndDesignations();
+	const consultResponseAndRepresent = new ConsultResponseAndRepresent();
+	const appealProcess = new AppealProcess();
+	const siteAccess = new SiteAccess();
+	const notifyParties = new NotifyParties();
+	const poReportAndSupportDocs = new PoReportAndSupportDocs();
+	let appealId;
+	let counter = 0;
+	cy.get(basePage?._selectors.trgovukTableRow).each(($row) => {
+		const rowtext = $row.text();
+		if (rowtext.includes(lpaManageAppealsData?.hasAppealType) && !rowtext.includes(lpaManageAppealsData?.todoInvalid)) {
+			if (counter === 0) {
+				cy.log(rowtext);
```

Comment:
Just checking you want to leave in this `cy.log`

Reply:
removed log